Cargo.toml

```diff
@@ -20,14 +20,13 @@ path = "src/lib.rs"
 
 [dependencies]
 base64 = "0.13"
-crypto-mac = "0.10"
 futures = { version = "0.3", optional = true }
-hmac = "0.10"
+hmac = "0.12"
 rand = "0.8"
 reqwest = { version = "0.11", features = ["blocking"] }
-sha-1 = "0.9"
+sha1 = "0.10"
 threadpool = "1.7"
-url = "1.7"
+form_urlencoded = "1"
```

Comment:
What's the motivation behind this dependency change?

Reply:
`url` v2 doesn't re-export `percent_encoding` anymore. I could have replaced `url` with `percent_encoding`, but seeing how all it was used for was to encode query strings I decided to go with `form_urlencoded` which is made specifically to deal with query strings.

All three crates are made by the same people, as you can see from the repo https://github.com/servo/rust-url